integration/test/plan/README.md

```diff
@@ -89,13 +98,20 @@ By default, this test is setup to run on a single compute node.  After
      *is* being invoked in it's default configuration which enables
      MPI support.  However, the JSON configuration for `geopmbench`
      specifies no MPI regions.  This means `geopmbench` will call
-     `MPI_Init` and `MPI_Finalize` but will not otherwise use MPI.
+     `MPI_Init` and `MPI_Finalize` but will not otherwise use MPI
+     (v3.1 vs. v3.2 differ on this; see below).
    - If the Runtime was built *without* MPI support, the
      `MPI_Init_thread`  region would not be present in the output
-     files.  `geopmbench` can be invoked with the GEOPMBENCH_NO_MPI
-     environment variable to avoid any MPI calls. For more information
+     files.  `geopmbench` can be invoked with the `GEOPMBENCH_NO_MPI`
+     environment variable to avoid MPI calls. For more information
      about the environment configuration of `geopmbench`, see
      [here](https://geopm.github.io/geopmbench.1.html#environment).
+     - `geopmbench ` @ v3.1 will always call `MPI_Init` and `MPI_Finalize`
```

Comment:
File a tech debt issue so that we remember to delete this comment on next release. Label it appropriately (next-release?)

Reply:
Done. https://github.com/geopm/geopm/issues/3723